### PR TITLE
Added virgil agent header

### PR DESCRIPTION
**Pull request by BuddahLD**

### CODE REVIEW

File: passw0rd/src/test/java/com/virgilsecurity/passw0rd/utils/VersionTestJava.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+import com.virgilsecurity.passw0rd.build.VersionVirgilAgent;
+import org.junit.jupiter.api.Test;
+
+/**
+ * VersionTestJava class.
+ */
+class VersionTestJava {
+
+  private static final String CURRENT_VERSION = "1.0.1";
```
Comment (andrii-iakovenko) [inline passw0rd/src/test/java/com/virgilsecurity/passw0rd/utils/VersionTestJava.java]: You can get version from build.gradle

File: passw0rd/src/test/kotlin/com/virgilsecurity/passw0rd/utils/VersionTest.kt [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+class VersionTest {
+
+    @Test fun version_test() {
+        val version = VersionVirgilAgent.VERSION
+
+        assertEquals(CURRENT_VERSION, version)
+    }
+
+    companion object {
+        private const val CURRENT_VERSION = "1.0.1"
```
Comment (andrii-iakovenko) [inline passw0rd/src/test/kotlin/com/virgilsecurity/passw0rd/utils/VersionTest.kt]: You can get version from build.gradle

File: passw0rd/src/test/kotlin/com/virgilsecurity/passw0rd/utils/VersionTest.kt [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    }
+}
```
Comment (andrii-iakovenko) [inline passw0rd/src/test/kotlin/com/virgilsecurity/passw0rd/utils/VersionTest.kt]: No empty line?

File: build.gradle [hunk 4]
```diff
@@ -40,7 +40,7 @@ buildscript {
             fuel          : '1.15.1',
 
             // Kotlin
-            kotlin        : '1.3.11',
+            kotlin        : '1.3.21',
```
Comment (andrii-iakovenko) [inline build.gradle:43]: Old Virgil SDK version used. See few lines above.

**Review by andrii-iakovenko — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 4 above)
Comment (BavovnaIsTaken) [reply build.gradle:43]: We're using com.virgilsecurity.crypto:common, com.virgilsecurity.crypto:foundation and com.virgilsecurity.crypto:phe which latest version is 5.0.1:
http://central.maven.org/maven2/com/virgilsecurity/crypto/phe/

**Review by andrii-iakovenko — APPROVED** [review]

**Pull request merged**